addons/l10n_in_edi_ewaybill/models/account_edi_format.py

```diff
@@ -78,8 +78,8 @@ def _l10n_in_edi_ewaybill_json_invoice_content(self, move):
     def _check_move_configuration(self, move):
         if self.code != "in_ewaybill_1_03":
             return super()._check_move_configuration(move)
-        base = self._l10n_in_edi_ewaybill_base_irn_or_direct(move)
         error_message = []
+        base = self._l10n_in_edi_ewaybill_base_irn_or_direct(move)
```

Comment:
@jva-odoo Is the change still needed then?

Reply:
@jco-odoo Not needed but I keep it because it has the same code in the same way. so any FW port PR does not have a conflict here.